_test/test/common/utils.dart

```diff
@@ -68,6 +68,9 @@ Future<void> _startServer(String command, List<String> buildArgs,
   }
 
   final proc = _process = await Process.start(command, buildArgs);
+  unawaited(proc.exitCode.then((_) {
```

Comment:
How did this come up? Was there a problem when the process died earlier than expected?